src/measure.cpp

```diff
@@ -1198,8 +1202,10 @@ int Measure::CastOffSystems(FunctorParams *functorParams)
     CastOffSystemsParams *params = vrv_params_cast<CastOffSystemsParams *>(functorParams);
     assert(params);
 
-    // Check if the measure has some overflowing control elements
-    int overflow = this->GetDrawingOverflow();
+    const bool hasCache = this->HasCachedHorizontalLayout();
+    int overflow = hasCache ? this->m_cachedOverflow : this->GetDrawingOverflow();
+    int width = hasCache ? this->m_cachedWidth : this->GetWidth();
+    int drawingXRel = this->m_drawingXRel;
```

Comment:
Why not use the cache for `drawingXRel`? Is this desired?

Reply:
It is (at least it should be?) the same because it had be restored by `Measure::HorizontalLayoutCache` at this stage.